deploy/k8s.yaml

```diff
@@ -40,6 +40,12 @@ rules:
   - delete
   - patch
   - watch
+- apiGroups:
```

Comment:
Please do not edit this file directly. You can edit deploy/kubernetes/base/* , and then `make deploy/k8s.yaml` .

Also, please did the same change in chart.

Reply:
done